heka/sandbox/decoders/extract_telemetry_dimensions.lua

```diff
@@ -81,6 +82,15 @@ local function sample(id, sampleRange)
     return hash.crc32(id) % sampleRange
 end
 
+function parse_creation_date(date)
+   local t = dt.rfc3339:match(date or "")
```

Comment:
Don't bother to try and parse if the data is missing. Please guard with something like `if not date or type(date) ~= "string" then return nil end`